src/runtime_src/core/edge/skd/sk_daemon.cpp

```diff
@@ -67,13 +67,9 @@ static void sigLog(const int sig)
 	while (std::getline (backtrace, bt_line, '\n') )
 	  xrt_core::message::send(severity_level::error, "SKD", bt_line );
 #endif
-	struct sigaction act = {};
-	act.sa_handler = SIG_DFL;
-	sigemptyset(&act.sa_mask);
-	act.sa_flags = 0;
-	sigaction(sig, &act, nullptr);
-	raise(sig);
-	exit(sig);
+	skd_inst->report_crash();
+	xrt_core::message::send(severity_level::error, "SKD", "SKD Reported crash!");
+	exit(128 + sig); // Convert to Linux exit code
```

Comment:
Please create a function to do this conversion.  It is not good to use magic number to do this type of exit.